src/db-service/shopping_cart.js

```diff
@@ -62,7 +62,7 @@ exports.getCart = async (cartId, format = true) => {
     let products =
       rows[0].id === null
         ? null
-        : rows.map(row => {
+        : rows.map((row) => {
             let price = row.price;
             total += price;
             return {
```

Comment:
I think instead of adding note and scopes, we should add a token option which has the same properties as the Create a new Authorization. If token is not set, we will start logging a warning with a URL to documentation which explains why, same as the token.noteUrl default

Reply:
Code LGTM. To me Gus's suggestion sounds best though. I am curious about use cases which could not be solved with the granular implementation. I am +1 to landing this as is if there are any other use cases and otherwise +0.